test/cql-pytest/conftest.py

```diff
@@ -88,16 +88,46 @@ def cql_test_connection(cql, request):
 def this_dc(cql):
     yield cql.execute("SELECT data_center FROM system.local").one()[0]
 
+@pytest.fixture(scope="session")
+def test_keyspace_tablets(cql, this_dc, has_tablets):
+    if not is_scylla(cql) or not has_tablets:
```

Comment:
Nitpick: I don't think the extra is_scylla() check is needed, has_tablets already is false if not scylla. And has_tablets is only calculated once while the is_scylla(cql) check takes more time.